### PR TITLE
[FIX] web: `pivotView` empty header cell background, adapt for milk

### PR DESCRIPTION
**Pull request by xlu-odoo**
Adjust pivot view to match new web client redesign: Set top-left empty header cell background-color to match view background (+ borders).

task-3329794
part of task-3326263

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by stefanorigano** [comment]
I think that the component name is `pivotView` not `Pivot view` ;)

**Comment by xlu-odoo** [comment]
@stefanorigano - Updated (green runbot) based on your desired result (your desired result had a white border, but I guess you wanted it to have 0 border at all)

File: addons/web/static/src/views/pivot/pivot_renderer.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
                                 <t t-set="isXAxis" t-value="true"/>
                                 <t t-set="isInHead" t-value="true"/>
                             </t>
-                            <th t-else="" t-att-colspan="cell.width" t-att-rowspan="cell.height" class="fw-normal border-start-0" t-att-class="{ 'border-top': cell_index != 0 }"/>
+                            <th t-else="" t-att-colspan="cell.width" t-att-rowspan="cell.height" class="border-0 fw-normal bg-100" t-att-class="{ 'border-0': cell_index != 0 }"/>
```
Comment (stefanorigano) [inline addons/web/static/src/views/pivot/pivot_renderer.xml:20]: Convention: classes order should match CSS declarations one ;)
https://www.odoo.com/documentation/master/developer/howtos/scss_tips.html#handling-utility-classes-verbosity

**Review by stefanorigano — APPROVED** [review]
@xlu-odoo , LTGM, just a convention remark.
Once fixed you can ping [@]pparidans for tech. review

 thanks 👍

**Comment by pparidans** [comment]
LGTM
PR is still [Draft], is it OK to be merged ?

**Comment by xlu-odoo** [comment]
> LGTM PR is still [Draft], is it OK to be merged ?

Yes 👍

**Comment by pparidans** [comment]
@robodoo r+

**Pull request closed without merging**